src/mlpack/methods/ann/layer/highway.hpp

```diff
@@ -171,6 +171,19 @@ class Highway
   //! Modify the gradient.
   OutputDataType& Gradient() { return gradient; }
 
+  //! Get the input width.
+  size_t const& InputWidth() const { return width; }
```

Comment:
Some modules in mlpack only check for `InputWidth` and expect `OutputWidth` is there as well, either we remove the getter here or add a pseudo `OutputWidth` method as well.

Reply:
I see!